go/vt/vttablet/heartbeat/reader.go

```diff
@@ -68,7 +68,7 @@ type Reader struct {
 }
 
 // NewReader returns a new heartbeat reader.
-func NewReader(checker connpool.MySQLChecker, config tabletenv.TabletConfig) *Reader {
+func NewReader(tsv connpool.TabletService, config tabletenv.TabletConfig) *Reader {
```

Comment:
I think this rename is a bit confusing. When I saw `TabletService`, I was thinking in something that would implement an RPC interface. Later down in the PR, I understood what's going on, but I think we should try to find a better name.